git-filter-repo

```diff
@@ -2903,10 +2903,10 @@ class RepoAnalyze(object):
 
     if os.path.isdir(reportdir):
       if args.force:
-        sys.stdout.write(_("Warning: Removing recursively: \"%s\"\n") % decode(reportdir))
+        sys.stdout.write(_("WARNING: Removing recursively: \"%s\"\n") % decode(reportdir))
         shutil.rmtree(reportdir)
       else:
-        sys.stdout.write(_("Error: dir already exists (use --force to delete): \"%s\"\n") % decode(reportdir))
+        sys.stdout.write(_("ERROR: dir already exists (use --force to delete): \"%s\"\n") % decode(reportdir))
```

Comment:
We have 29 messages starting with "Error", and none starting with "ERROR" in the current codebase.  I think it feels weird to change 1 of them.

"Warning" is a bit different.  We only have three, with one being all-caps.  But, for consistency, I'd say we should probably change the all-caps one to a single capital, and then not make this change.